packages/experiments/src/components/Microfeedback/Microfeedback.view.tsx

```diff
@@ -0,0 +1,178 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { withSlots } from '../../Foundation';
+import { Stack } from '../../Stack';
+import { IconButton, IButtonStyles, DefaultButton } from 'office-ui-fabric-react/lib/Button';
+import { IStackStyles } from 'office-ui-fabric-react/lib/Stack';
+import { Callout } from 'office-ui-fabric-react/lib/Callout';
+import { FocusZone, FocusZoneDirection } from 'office-ui-fabric-react/lib/FocusZone';
+import { List } from 'office-ui-fabric-react/lib/List';
+
+import { IMicrofeedbackComponent, IMicrofeedbackProps } from './Microfeedback.types';
+import { getTheme, mergeStyleSets, FontWeights } from 'office-ui-fabric-react/lib/Styling';
+import { IMicrofeedbackState } from './Microfeedback.state';
+
+import { initializeIcons } from '@uifabric/icons';
+
+const theme = getTheme();
+
+const microfeedbackStyles: IStackStyles = {
+  root: [
+    {
+      margin: 8,
+      float: 'right'
+    }
+  ]
+};
+
+const microfeedbackItemStyles: IButtonStyles = {
+  root: [
+    theme.fonts.small,
+    {
+      width: '100%',
+      color: theme.palette.neutralPrimary,
+      fontWeight: FontWeights.light,
+      fontSize: 12
+    }
+  ]
+};
+
+const styles = mergeStyleSets({
+  title: [
+    theme.fonts.medium,
+    {
+      margin: 8,
+      color: theme.palette.neutralPrimary,
+      fontWeight: FontWeights.semibold
+    }
+  ],
+  body: [
+    theme.fonts.small,
+    {
+      margin: 6,
+      color: theme.palette.neutralPrimary,
+      fontWeight: FontWeights.regular
+    }
+  ]
+});
```

Comment:
We should convert these all to styles functions that take in the theme as part of the props as an argument. Our [Text component styling](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Text/Text.styles.ts) will be similar to what you can do here.

Reply:
Fixed